### PR TITLE
update zh_TW translation

### PR DESCRIPTION
**Pull request by OriginCode**

### CODE REVIEW

File: po/zh_TW.po [hunk 1]
```diff
@@ -1,883 +1,160 @@
...
+#: prefs.js:930 prefs.js:1023 prefs.js:1260 prefs.js:1344 prefs.js:1455
+#: prefs.js:1489 prefs.js:1531
 msgid "Reset to defaults"
 msgstr "恢復預設值"
 
-#: prefs.js:431
+#: prefs.js:460
 msgid "Default (Primary monitor)"
-msgstr "主顯示器"
+msgstr "默認（主顯示器）"
```
Comment (pan93412) [inline po/zh_TW.po]: `預設` is better.

File: po/zh_TW.po [hunk 2]
```diff
@@ -1,883 +1,160 @@
...
-#: prefs.js:1052
+#: prefs.js:1253
 msgid "Ungrouped application options"
 msgstr "未分組的應用程式選項"
 
-#: prefs.js:1136
+#: prefs.js:1337
 msgid "Customize middle-click behavior"
-msgstr "自定義中鍵點選行為"
+msgstr "自定義滑鼠中鍵點選行爲"
```
Comment (pan93412) [inline po/zh_TW.po]: `自訂` is better.

File: po/zh_TW.po [hunk 3]
```diff
@@ -1,883 +1,160 @@
...
 
-#: prefs.js:1247
+#: prefs.js:1448
 msgid "Advanced hotkeys options"
-msgstr "高階熱鍵選項"
+msgstr "進階熱鍵選項"
```
Comment (pan93412) [inline po/zh_TW.po]: `hotkeys` -> `快捷鍵`

File: po/zh_TW.po [hunk 4]
```diff
@@ -1,883 +1,160 @@
...
+msgid "Event logs"
+msgstr "事件日誌"
+
+#: appIcons.js:1588
+msgid "System"
+msgstr "系統"
+
+#: appIcons.js:1593
+msgid "Device Management"
+msgstr "設備管理"
```
Comment (pan93412) [inline po/zh_TW.po]: `裝置管理`

**Comment by jderose9** [comment]
Thanks so much, @OriginCode . It looks like you may have inadvertently removed quite a lot of the messages though in c46b0eb1a6eba2ab16e39d533ca60099a937523a ?

**Comment by OriginCode** [comment]
@jderose9 Ah-huh, I don't know why when I generated the pot file, it didn't include these missing parts.

**Comment by jderose9** [comment]
@OriginCode will you be submitting soon? I was going to wait until you resubmitted to release, but was hoping to do so soon. Thanks!

**Comment by OriginCode** [comment]
> @OriginCode will you be submitting soon? I was going to wait until you resubmitted to release, but was hoping to do so soon. Thanks!

Well, maybe I will recommit in a week.

**Pull request closed without merging**